lms/models/assignment.py

```diff
@@ -49,6 +50,20 @@ class Assignment(BASE):
         nullable=False,
     )
 
+    is_gradable = sa.Column(
+        sa.Boolean(),
+        default=False,
+        server_default=sa.sql.expression.false(),
+        nullable=False,
+    )
+    """Whether this assignment is gradable or not."""
```

Comment:
Should this be `gradable` instead? Our other boolean columns don't have `is_` on the front?